docs/source/API/containers/Unordered-Map.rst

```diff
@@ -39,6 +39,12 @@ Description
 
       .. warning:: Host Only
 
+   .. cpp:function:: UnorderedMap(const ALLOC_PROP &prop, uint32_t capacity_hint);
+
+      Create map using the properties with enough space for at least capacity_hint number of objects
+
+      .. warning:: Host Only
+
```

Comment:
Can you add documentation for alloc props? Maybe adding something about SequentialHostInit adding allowing the map to store views.

If this was added in 4.7 should do something like:

```suggestion
   .. cpp:function:: UnorderedMap(const ALLOC_PROP &prop, uint32_t capacity_hint);

      Create map using the properties with enough space for at least capacity_hint number of objects

      .. warning:: Host Only

      .. versionadded:: 4.7
```